dist/index.html

```diff
@@ -35,18 +36,77 @@
         })
       }
     </script>
+
     <link rel="icon" type="image/png" href="favicon-32x32.png">
   </head>
+
   <body>
     <div class="content" style="position: absolute; width: 100vw; height: 100vh; top: 0; left: 0; overflow: hidden; background: black; margin: 0; padding: 0;"></div>
     <script type="text/javascript" src="itkVtkViewer.js">
     </script>
     <script>
-      var container = document.querySelector('.content');
-      var viewer = itkVtkViewer.processURLParameters(container, { fullscreen: true });
+     async function setupViewerForImJoy(container) {
+      const imjoyRPC = await imjoyLoader.loadImJoyRPC({
+        api_version: '0.2.3'
+      })
+      const version = itkVtkViewer.version
+      const api = await imjoyRPC.setupRPC({
+        name: 'itk-vtk-viewer',
+        version,
+        description: '2D / 3D web image, mesh, and point set viewer using itk.js and vtk.js ',
+        type: 'rpc-window',
+        docs: 'https://kitware.github.io/itk-vtk-viewer/docs/imjoy.html',
```

Comment:
Please also consider adding the following keys:
 * `authors`: a list of author names
 * `license`: name of the license
 * `labels`: e.g. ["visualization"]
 * `icon`: url to an small squared icon image in png/jpg format, size < 100x100px
 * `cover`: url to one or a list of cover images, typically a screenshot or a short gif which will show in the plugin store and the plugin installation dialog.

I think if we already have some images used elsewhere, `icon` and `cover` are recommended since they will show up in the plugin menu and the installation dialog.

Reply:
Thanks for the tips! Added.